### PR TITLE
Updated domains for ContentSquare and added Jaywing

### PR DESCRIPTION
**Pull request by rockeynebhwani**

### CODE REVIEW

**Comment by rockeynebhwani** [comment]
This is my first open source contribution.. Updating some domains and a new third party which we found on one of the sites I manage

File: data/entities.json5 [hunk 1]
```diff
@@ -11273,4 +11273,12 @@
     "domains": ["*.auth0.com"],
     "examples": ["cdn.auth0.com"]
   }
+  {
+    "name": "Jaywing",
+    "homepage": "https://jaywing.com/",
+    "categories": ["marketing"],
+    "domains": ["*.almanac.jaywing.com"],
```
Comment (zeman) [inline data/entities.json5]: I think it's better to just wildcard the top level domain. eg *.jaywing.com

File: data/entities.json5 [hunk 2]
```diff
@@ -9508,7 +9508,7 @@
   {
     "name": "Urban Airship",
     "categories": ["ad"],
-    "domains": ["*.urbanairship.com"]
+    "domains": ["*.urbanairship.com", "aswpsdkus.com"]
```
Comment (zeman) [inline data/entities.json5]: Wildcarding *.aswpsdkus.com will protect against any changes.

Comment (rockeynebhwani) [reply data/entities.json5]: Thanks @zeman. Updated. Also, made following changes - 1)  "Urban Airship" got re-branded as just "Airship" recently so updated the name 2) I think "ad" was wrong category for this third party, "marketing" is more suited.

File: data/entities.json5 (hunk 1 above)
Comment (rockeynebhwani) [reply data/entities.json5]: updated

**Pull request merged**